### PR TITLE
Refactor pipeline strings to tuples

### PR DESCRIPTION
**Pull request by lmdehaas**
#10

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-0.1.2
+1.0.0
```
Comment (anneschuth) [inline VERSION]: @textpipe/developers any opinions on this? we're breaking backwards compatibility, so I guess this should indeed be bumped?

Comment (dodijk) [reply VERSION]: I don't think we need to break backward compatibility for this. We can just accept either a tuple or a string as an element in a pipeline.

File: textpipe/pipeline.py [hunk 2]
```diff
@@ -30,10 +31,11 @@ def __init__(self, operations, language=None, hint_language=None, **kwargs):
         self.kwargs = kwargs
         self._operations = []
         # loop over pipeline operations and instantiate operation classes.
-        for operation_name in operations:
-            oper_cls = getattr(textpipe.operation, operation_name)
-            # todo: pass in config to operation constructor, i.e., oper_cls(config)
-            self._operations.append(oper_cls())
+        for oper_tuple in operations:
+            oper_name = oper_tuple[0]
+            oper_kwargs = oper_tuple[1] if len(oper_tuple) > 1 else {}
```
Comment (dodijk) [inline textpipe/pipeline.py]:
```python
for operation in operations:
    oper_name = operation if type(operation) == str else operation[0]
    oper_kwargs = operation[1] if type(operation) != str and len(operation) > 1 else {}
```

**Comment by anneschuth** [comment]
image I didn't actually do that, not sure what this is

**Pull request merged**